ubuntu.md

```diff
@@ -0,0 +1,22 @@
+### Ubuntu 16.04
+Instructions for installing VirtualBox & Vagrant on Ubuntu 16.04
+
+TODO: test this procedure
+```shell
+# TODO create ~/.ssh/rhce and ssh-add
+wget -q -O - http://download.virtualbox.org/virtualbox/debian/oracle_vbox_2016.asc | sudo apt-key add -
+sudo sh -c 'echo "deb http://download.virtualbox.org/virtualbox/debian `lsb_release -sc` non-free contrib" > /etc/apt/sources.list.d/virtualbox.org.list'
+sudo apt-get update
+sudo apt-get install dkms
```

Comment:
we can reduce the lines by put package names in one command:
`sudo apt-get install git dkms virtualbox-5.1`